src/mono/wasm/runtime/js_support.js

```diff
@@ -0,0 +1,25 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// THIS FILE IS COPIED DIRECTLY INTO THE DOTNET.JS FILE WITHOUT BEING RUN OR COMPILED/OPTIMZIED
+// IT I MEANT AS A SET OF JS TOOLS TO SUPPORT THE SDK
+
+var JSSupportLib = {
+    // Loads the config file located in the root of the project
+    load_config: function (callback) {
+        if (ENVIRONMENT_IS_NODE){
+            const config = require('./mono-config.json');
+            callback(config);
+        }
+        var xobj = new XMLHttpRequest();
+        xobj.overrideMimeType("application/json");
+        xobj.open('GET', './mono-config.json', true);
+        xobj.onreadystatechange = function () {
+            if (xobj.readyState == 4 && xobj.status == "200") {
+                const config = JSON.parse(xobj.responseText);
+                callback(config);
+            }
+        };
+        xobj.send();  
```

Comment:
Is this meant for non-node cases?

Reply:
Yes to keep the node and browser implementations similar I use this code to load for browser and the require for node.

Reply:
Should the xhr stuff be in a `else { .. }` then?